patroni/ctl.py

```diff
@@ -1372,3 +1373,246 @@ def format_pg_version(version):
         return "{0}.{1}.{2}".format(version // 10000, version // 100 % 100, version % 100)
     else:
         return "{0}.{1}".format(version // 10000, version % 100)
+
+
+def enrich_config_from_running_instance(config: Dict[str, Any], no_value_msg: str, dsn: Optional[str] = None) -> None:
+    from getpass import getuser
+    from patroni.postgresql.config import parse_dsn
+    from patroni.config import AUTH_ALLOWED_PARAMETERS_MAPPING
+
+    su_params = parsed_dsn = {}
```

Comment:
This way both variables will point to the same dict object.
Currently it doesn't create any problem only because parsed_dsn is either empty or overwritten by the result of parse_dsn() call.
I would suggest to declare objects on separate lines:
```suggestion
    su_params = {}
    parsed_dsn = {}
```